test/transport/core.test.js

```diff
@@ -280,23 +281,23 @@ test('pino.transport with an array including a pino-pretty destination', async (
       }
     }]
   })
-  teardown(transport.end.bind(transport))
+  t.after(transport.end.bind(transport))
   const instance = pino(transport)
   instance.info('hello')
   await Promise.all([watchFileCreated(dest1), watchFileCreated(dest2)])
   const result1 = JSON.parse(await readFile(dest1))
   delete result1.time
-  same(result1, {
+  t.assert.deepStrictEqual(result1, {
     pid,
     hostname,
     level: 30,
     msg: 'hello'
   })
   const actual = (await readFile(dest2)).toString()
-  match(strip(actual), /\[.*\] INFO.*hello/)
+  match(strip(actual), /\[.*\] INFO.*hello/, t)
```

Comment:
I believe we can use the standard `t.assert.match` here. This is matching a regular expression against a string.

Reply:
Done